### ISSUE TITLE
Component connect multiple consumers

### ISSUE THREAD

**Issue by idarbuashvili** [Contiamo/react-connect-context#6]
Hi, I believe this library can be further improved by adding possibility to consume multiple consumers at once. I can provide code for that, but not in TS in JS. Thanks

**Comment by TejasQ** [comment]
Sure! If you'd like to start with providing JS, we can turn it into TS and collaborate if you're up for it. 💞

**Comment by idarbuashvili** [comment]
Yes, sure. https://gist.github.com/idarbuashvili/a57ca6b1ea86d4828f99ea21c14d246e, I believe the gist code above does exactly, what is now on the repo, but additionally I added recursive approach to handle multiple consumers passed in as rest params.

**Comment by TejasQ** [comment]
Wanna turn that into a PR? 😄

**Comment by idarbuashvili** [comment]
Sure, will do later today or tomorrow :grin:

**Comment by idarbuashvili** [comment]
Maybe as well turn into TS
